### DISCUSSION TITLE
Help needed polybar showing both workspace Icon and Numbers

### DISCUSSION THREAD

**Discussion by ghost** [polybar/polybar#3114] [Q&A] [answered]
What im trying to do i only have the icons visible and not the workspace numbers. Ive been starring myself blind at this problem, and ive tried all the fixes i could find and non working so far

polybar config:

[module/i3]
type = internal/i3
format = <label-state> <label-mode>
index-sort = true
strip-wsnumbers = true
pin-workspaces = true

ws-icon-0 = 1;󰅴
ws-icon-1 = 2;󰈹
ws-icon-2 = 3;󰓇
#ws-icon-3 = "4;"
#ws-icon-4 = "5;"
ws-icon-default = 󰜌 
#󰨑 󱗿 󱥸

label-active = %icon%
label-active-background = ${colors.bg-alt}
label-active-foreground = ${colors.fg}
label-active-underline= ${colors.primary}
label-active-padding = 2

label-occupied = %icon%
label-occupied-padding = 2

label-urgent = %icon%
label-urgent-background = ${colors.alert}
label-urgent-padding = 2

label-empty = %icon%
label-empty-foreground = ${colors.disabled}
label-empty-padding = 2

i3 config:

set $ws1 "1"
set $ws2 "2"
set $ws3 "3"
set $ws4 "4"
set $ws5 "5"
set $ws6 "6"
set $ws7 "7"
set $ws8 "8"
set $ws9 "9"
set $ws10 "10"

![help](https://github.com/polybar/polybar/assets/167023058/6d9fa8e1-8d62-4c77-8734-f1bfabaa4fa9)

**Comment by patrick96** [comment, marked as answer]
Almost all of those label names are wrong and don't exist in the i3 module: https://github.com/polybar/polybar/wiki/Module:-i3

**Reply by ghost** [reply]
God i must be low iq, i looked at that page for 4 hours yesterday. sorry
